File: src/main/java/com/example/climbingapi/model/Wall.java
package com.example.climbingapi.model;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.math.BigDecimal;
import java.time.OffsetDateTime;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class Wall {

    private Integer id;
    private Integer areaId;
    private String name;
    private String description;
    private BigDecimal latitude;
    private BigDecimal longitude;
    private String approachInfo;
    private OffsetDateTime createdAt;
}

File: src/main/kotlin/com/example/climbingapi/model/Wall.kt
package com.example.climbingapi.model

import java.math.BigDecimal
import java.time.OffsetDateTime

data class Wall(
    val id: Int?,
    val areaId: Int?,
    val name: String?,
    val description: String?,
    val latitude: BigDecimal?,
    val longitude: BigDecimal?,
    val approachInfo: String?,
    val imageKey: String?,
    val createdAt: OffsetDateTime?,
    val optimizedKey: String? = null,
    val thumbnailKey: String? = null
)
